.github/workflows/iroha2-dev-pr-wasm.yaml

```diff
@@ -12,6 +12,10 @@ on:
       - 'wasm/**.json'
       - 'wasm/**.toml'
       - 'wasm/**.yml'
+      - 'wasm/validator/**.rs'
+      - 'wasm/validator/**.json'
+      - 'wasm/validator/**.toml'
+      - 'wasm/validator/**.yml'
```

Comment:
don't previous `**` capture all this already?

Reply:
Indeed